src/Selda/PG.purs

```diff
@@ -72,18 +106,14 @@ insert (Table { name }) xs = concat <$> traverse insert1 xs
       cols = joinWith ", " $ tableColumnNames (RLProxy ∷ RLProxy rl)
       xTup = hfoldl RecordToTuple unit x
       xLen = hfoldl RecordLength 0 x
-      placeholders = 
+      placeholders =
         Array.range 1 xLen # map (\i → "$" <> show i) # joinWith ", "
-      q_str = 
+      qStr =
```

Comment:
Sorry for messing up coding style convention. I can revert this change. Do you use this "snake style" here on purpose?